_candidates_en/IppeiTakahashi_1.html

```diff
@@ -0,0 +1,19 @@
+---
+name: Ippei Takahashi
+title: Functional server side development by Play, Slick, Scalaz
+length: 40
+audience: Intermediate
+language: Japanese
+twitter: 
+github: ippei-takahashi
+icon: http://gravatar.com/avatar/e49eefa4110fda959f19da1341871b53
+organization: Recruit Marketing Partners Co., Ltd.
+---
+<p>
+In our team, we use Play framework, Slick, and Scalaz for new product's server side implementation.
+Slick after version 3.0 returns DB operation result by Future. And it can use Either for error handling. They are convenient, but they could be negative factor for code's readability. Monad and Monad transformer provided by Scalaz are useful for data operation wrapped by context(Future, Either etc..), and they aid high readable code what uses "for" operator.
```

Comment:
原文: バージョン3.0以降のSlickは、DB操作の結果をFutureに包んで返します。また、エラー処理にはEitherが使えます。
訳案: Slick 3.0 and above returns the result of DB operations wrapped in a `Future`, and it can also use `Either` for the error handling.